test/helpers/errors.go

```diff
@@ -0,0 +1,29 @@
+package helpers
+
+import "fmt"
+
+// NewSSHMetaError returns a new SSHMetaError with the given string and a
+// callback function
+func NewSSHMetaError(s string, callback func() string) *SSHMetaError {
+	return &SSHMetaError{
+		s:       s,
+		errorCB: callback,
+	}
+}
+
+// SSHMetaError is a custom error that executes a callback function each time
+// that it's called Error() function
```

Comment:
Documentation is not a valid phrase; "executes a callback function when its Error() function is invoked" ?